### DISCUSSION TITLE
How to Export Original Shaders Without Needing UABE or Other Tools for Replacement?

### DISCUSSION THREAD

**Discussion by Muruaaki** [AssetRipper/AssetRipper#1383] [Q&A] [answered]
I am preparing to extract an switch archive of version 5.6.5p1 and 2021.3.19f1. I chose the YAML shader, but it cannot be opened by Unity.
The dummy shader can be opened by Unity, but how should I set it up so that when exporting, I don't need to use UABE or other software to replace the shader again?

**Comment by Muruaaki** [comment]
[phcw1105_f01a.zip](https://github.com/user-attachments/files/15724002/phcw1105_f01a.zip)
I'm not sure if I can upload the archive, but this is the archive.

**Comment by ds5678** [comment, marked as answer]
There's nothing to be done. That's just how it is.
